app/components/road-sign-form/shape-manager.ts

```diff
@@ -1,7 +1,7 @@
 import { service } from '@ember/service';
 import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
-import type Store from 'ember-data/store';
```

Comment:
can you revert this again :smile:

Reply:
This actually was a lint error: `Imports from @ember-data packages should be preferred over imports from ember-dataeslint[ember/use-ember-data-rfc-395-imports](https://github.com/ember-cli/eslint-plugin-ember/tree/master/docs/rules/use-ember-data-rfc-395-imports.md)`

Reply:
Ah, ok, I thought this was your temporary commit :smile:

In any case, that rule is outdated, I think we ignore it in some other places. I fixed the issue in the linter package, but we need to update the package to get that fix (which we should probably do in a separate PR).

https://github.com/ember-cli/eslint-plugin-ember/blob/master/CHANGELOG.md#bug-bug-fix